django_x509/base/models.py

```diff
@@ -142,6 +143,10 @@ def clean_fields(self, *args, **kwargs):
         # must be done here in order to validate imported fields
         # and fill private and public key before validation fails
         if not self.pk and self.certificate and self.private_key:
+            try:
+                crypto.load_certificate(crypto.FILETYPE_PEM,self.certificate)
+            except OpenSSL.crypto.Error as e:
+                raise ValidationError(_('Start line missing in Certificate'))
```

Comment:
We cannot be sure of what is the cause of the error, it would be better to pass the exception message instead of an arbitrary chosen message, something like:

`raise ValidationError(_('OpenSSL error: {0}').format(e.message))`

Would do. Not sure of the syntax though, I haven't tried it. Just writing it from the top of my head.